### PR TITLE
STIJ-355: Fixed background accolade in contact-details context.

### PR DESCRIPTION
**Pull request by jeroendegloire**
## Description
Updated scss of contact details component.

## Motivation and Context
Add an inverted background for accolade component in the context of contact-details highlight block.

## How Has This Been Tested?
Manual tested in KotAtGent.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

**Review by CaliJens — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this style guide are documented here.
 
+## [3.0.3]
```
Comment (delrueba) [inline CHANGELOG.md]:
```suggestion
## [Unreleased]
```

**Review by delrueba — CHANGES_REQUESTED** [review]

File: components/31-molecules/contact-details/_contact-details.scss [hunk 2]
```diff
@@ -32,6 +32,13 @@ $avatar-width: 14rem;
         box-shadow: none;
       }
     }
+
+    .accolade {
+      &::before,
+      &::after {
+        @include theme('background', 'color-secondary', 'highlight-background');
```
Comment (delrueba) [inline components/31-molecules/contact-details/_contact-details.scss]:
```suggestion
        @include theme('background', 'color-secondary', 'contact-details-inverted-background-color');
```

**Review by delrueba — CHANGES_REQUESTED** [review]

File: components/31-molecules/highlight/_highlight.scss [hunk 3]
```diff
@@ -35,7 +35,7 @@
   .accolade {
     &::before,
     &::after {
-      @include theme('background', 'color-primary--lighten-5', 'highlight-background');
+      @include theme('background', 'color-primary--lighten-5', 'highlight-inverted-background');
```
Comment (delrueba) [inline components/31-molecules/highlight/_highlight.scss]: No need to change this one.
(it's also quite dangerous, as you'll potentially break existing themes)

I've checked and 'inverted' only exists in the contact-details, not in highlight itself.

```suggestion
      @include theme('background', 'color-primary--lighten-5', 'highlight-background');
```

**Review by delrueba — CHANGES_REQUESTED** [review]

File: components/31-molecules/highlight/_highlight.scss (hunk 3 above)
Comment (jeroendegloire) [reply components/31-molecules/highlight/_highlight.scss]: Ah I see, I changed highlight file in place of contact details 🤦 .

File: components/31-molecules/contact-details/_contact-details.scss (hunk 2 above)
Comment (delrueba) [reply components/31-molecules/contact-details/_contact-details.scss]: Still need to commit this suggestion (if you agree of course).
You don't want 'highilight-background' --> that would be a lighter shade of color primary

You want 'contact-details-inverted-background-color' --> white.

You can even use the same themify function on line 27 to make sure they are both the same color.

**Review by delrueba — APPROVED** [review]

**Pull request merged**